include/RAJA/policy/simd/forall.hpp

```diff
@@ -9,6 +9,10 @@
  *          These methods should work on any platform. They make no
  *          asumptions about data alignment.
  *
+ *          Note: SIMD reductions under nested parallelism are 
```

Comment:
This comment is a little unclear. Isn't the point that one cannot use reductions in forall methods with simd execution policies and expect the code to work correctly? This is not the same as nested parallelism.

Reply:
I thought the issue was that if a reduction operation is invoked in vector code, then the results are not correct because we don't have a way to record the reduction for each simd lane?